deploy/site-config.sh

```diff
@@ -12,9 +12,14 @@ dump_config() {
 	site="$2"
 	vo="$3"
 	oidc_token="$4"
+	set -x
+	fedcloud openstack token issue \
+		--oidc-access-token "$oidc_token" \
+		--site "$site" --vo "$vo" -j
+	set +x
```

Comment:
This means printing the openstack token in the logs, right?

The command returns:

```json
[
{
  "Site": "SITE",
  "VO": "VO",
  "command": "token issue",
  "Exception": null,
  "Error code": 0,
  "Result": {
    "expires": "2025-04-15T09:04:49+0000",
    "id": "blah",
    "project_id": "blah",
    "user_id": "blan"
  }
}
]
```

Therefore I propose to check on `Error code` before moving on.

Reply:
Indeed the `set -x` was not a good idea. About checking the error code, it would improve the error reporting, but at the end would not change things much no?